langs/awk/Dockerfile

```diff
@@ -2,21 +2,21 @@ FROM alpine:3.20 AS builder
 
 RUN apk add --no-cache build-base curl
 
-RUN curl https://ftp.gnu.org/gnu/gawk/gawk-5.3.0.tar.xz | tar xJ
+RUN curl https://ftp.gnu.org/gnu/gawk/gawk-5.3.0.tar.xz \
+  | tar xJ --strip-components 1
 
-RUN mv gawk-* gawk      \
- && cd gawk             \
- && ./configure         \
-   --disable-lint       \
-   --prefix /usr        \
- && make install        \
+RUN ./configure    \
+    --disable-docs \
+    --disable-lint \
+    --prefix=/usr  \
+ && make -j`nproc` \
+ && make install   \
  && strip /usr/bin/gawk
 
 FROM codegolf/lang-base-no-proc
 
-COPY --from=0 /usr/bin/gawk            /usr/bin/gawk
-COPY --from=0 /usr/lib/gawk/           /usr/lib/gawk/
 COPY --from=0 /lib/ld-musl-x86_64.so.1 /lib/
+COPY --from=0 /usr/bin/gawk            /usr/bin/
 
```

Comment:
I wonder if you can retest the solutions, even just partially, without deployment. If shit breaks, it's just a matter of restoring the transfer of `/usr/lib/gawk`. I did test the defaults though, no complaints.

Reply:
Tried producing a static binary as well, but that wasn't working out quick enough for some odd reason.